### PR TITLE
#158 support v1 story table (after #185)

### PR DESCRIPTION
**Pull request by plocket**
#158. I think this will fully support v1 tables, though it is a bit awkward and its testing isn't super thorough yet because we don't have even a single signature page in the online interviews. Once that's done and added to the tests, I think we can be fairly confident this will,  except for the primitive proxy var (x, i, j...) setting,  support everything that the we were previously able to support.

Next I think we need to add unit tests to workflows and get started on proxy vars themselves.

### CODE REVIEW

**Comment by niharikasingh** [comment]
I changed the base branch to remove duplicate commits from the diff, but once you merge the other 158 branch into `main`, you can change this MR's base branch to `main`.

File: lib/steps.js [hunk 1]
```diff
@@ -183,18 +183,26 @@ When(/^the user gets to ?(?:the)?(?: ?question)?(?: id)? "([^"]+)" with this dat
...
   for ( let row of var_data ) {
-    // We need to take care of this before saving this row in the report
-    row.var = row.var || 'no alkiln story table value';
+    let result = {
+      original: row,
+      // Give fake data if needed to prevent some kind of hidden INPUT field from being found
+      // See bug at https://github.com/plocket/docassemble-cucumber/issues/79
+      // May be able to remove this once converted to `getPageData`
+      var: row.var || 'al_no_var_name',
+      value: row.choice || row.value,
```
Comment (niharikasingh) [inline lib/steps.js:202]: Should this be the other way around? The old table headers are `var/value/checked` but here the old table's header appears second.

**Review by niharikasingh — APPROVED** [review]

File: lib/steps.js (hunk 1 above)
Comment (plocket) [reply lib/steps.js:202]: I think you're totally right, wow. Thanks for spotting that.

Comment (plocket) [reply lib/steps.js:202]: I think this needs more changes. `.checked` should be `.checked` or `.choice` to get the true/false values. I think.

Comment (plocket) [reply lib/steps.js:202]: We need to write test cases for these, though I can't think of what they would be atm.

**Review by niharikasingh — CHANGES_REQUESTED** [review]
Need to figure out how to combine old and new tables.

File: lib/steps.js (hunk 1 above)
Comment (plocket) [reply lib/steps.js]: Created a test in the end and it tests some things ¯\_(ツ)_/¯

File: tests/unit_tests/tables.fixtures.js [hunk 2]
```diff
@@ -82,4 +82,283 @@ tables.buttons_event_action = [
 ];
 
 
+// =================
+// Converting from old table to new table
+// =================
+tables.original_formatting = [
```
Comment (niharikasingh) [inline tests/unit_tests/tables.fixtures.js:88]: The amount of repetition in this test setup makes me slightly nervous that copy+paste errors while writing the test could result in spurious test failures. Is it possible to write tests where the input is just the original table (as the developer would write it) and it gets compared to the processed version of the new-style table (again, as the developer would write it)?

**Review by niharikasingh — APPROVED** [review]

File: tests/unit_tests/tables.fixtures.js (hunk 2 above)
Comment (plocket) [reply tests/unit_tests/tables.fixtures.js:88]: That's a complex question. Created #188 for it.

**Pull request merged**